### PR TITLE
[Redesign] Training Screen + Accessibility (fixes - home/language screen)

### PR DESCRIPTION
**Pull request by nayushi**
- Redesign Training Screen
- Training Screen Accessibility
- Fixes Accessibility bugs at Home and Add new language screen
- Removed emoji
- Added empty scenario at home and training screen

### CODE REVIEW

File: VocabularyTrainer/Home Screen/ViewModel/HomeViewModel.swift [hunk 1]
```diff
@@ -20,10 +20,8 @@ final class HomeViewModel {
         if let languages = UserDefaults.standard.array(forKey: UserDefaultKeys.languages) as? [String] {
             retVal = languages.map {
```
Comment (misteu) [inline VocabularyTrainer/Home Screen/ViewModel/HomeViewModel.swift:21]: For some reason, existing languages are not loaded anymore.

When setting a breakpoint here (line 21) on `master` it's stopping after the `if let` with the languages being there. But not on this branch for some reason they are not loaded anymore 🤷 

I don't really get why though when looking at it. It does not look as if you renamed the keys used for loading the user default values or something.

Do you have a guess what might be the issue?

**Comment by misteu** [comment]
I created another branch from this PR's branch where I fixed merge conflicts: https://github.com/misteu/VocabularyTraining/tree/feature/fix-merge-conflict-for-pr

The already released fixes for the import/export feature where not on master yet due to ssh issues on my machine last time.

If everything seems fine, just merge this branch into your branch (should be possible without further conflucts I hope 🤞)

File: VocabularyTrainer/Home Screen/ViewModel/HomeViewModel.swift (hunk 1 above)
Comment (misteu) [reply VocabularyTrainer/Home Screen/ViewModel/HomeViewModel.swift:21]: ah haha 😄 

probably this is causing it: 
image

Please change this back to the original bundle-id as well :)

Comment (nayushi) [reply VocabularyTrainer/Home Screen/ViewModel/HomeViewModel.swift:21]: Done

**Comment by misteu** [comment]
One last thing:

The "take a look" does work now differently. Before you could take a look (i.e. unhide the word) and tap whether you guessed correctly or not. Now it seems to be unhidden as long as you hold it but you still have to type the other side.

I think we should keep it like before as I only added this because of user feedback back then. It's main purpose was to quickly being able to go through learning without having to type.

**Pull request merged**